Dockerfile

```diff
@@ -7,18 +7,22 @@ RUN apt-get update && \
     wget \
     curl \
     unzip \
-    default-jre
+    default-jre\
+    dos2unix
 
 # Download and extract Axon.ivy Engine
 RUN wget http://download.axonivy.com/7.0.0/AxonIvyEngine7.0.0.55921_All_x64.zip -O AxonIvyEngine7.zip && \
     unzip AxonIvyEngine7.zip -d /opt/AxonIvyEngine7 && \
     rm -f AxonIvyEngine7.zip && \
     useradd -s /sbin/nologin axonivy && \
-    chown -R axonivy:axonivy /opt/AxonIvyEngine7 && \
-    chown -R axonivy:axonivy /opt/AxonIvyEngine7/
+    chown -R axonivy:axonivy /opt/AxonIvyEngine7
 
 COPY start-axonivy-engine.sh /usr/local/bin/start-axonivy-engine.sh
 
+# Convert line ending from windows style to unix
+# since if you build the image on Windows and then execute, unix wont be able to find the sh file
+RUN dos2unix /usr/local/bin/start-axonivy-engine.sh
```

Comment:
It is really necessary to do this? I mean, finally we just copy a file from the local filesystem to the docker image. Since the file in GIT doesn't have Windows style line endings it should work. Except you have some (git-)settings which automatically change the line-endings during clone... (https://help.github.com/articles/dealing-with-line-endings/#platform-windows)

Reply:
Without this part the Dockerfile works on Unix-based system. However if you build the Dockerfile on Windows using `docker build` then `docker run` you will get error "start-axonivy-engine.sh not found". I got that problem and fixed it according to this article http://willi.am/blog/2016/08/11/docker-for-windows-dealing-with-windows-line-endings/.

Reply:
Yeah, I also saw this article. But it also mentions a second solution, setting `core.autocrlf=input` when cloning.
I'm not an expert in this topic. So, just that I got this right: When cloning a git repo in Windows, git will automatically convert LF to CRLF in your local copy. And that's why your local built image is failing. If that's right, I'd prefer to use the `core.autocrlf=input` option, since that works for all files. If we use `dos2unix` we really have to take care that we use it for all scripts copied into the image...
What do you think?

Reply:
Yeah, I was be able to pull engine 6 from your Docker Hub without dos2unix so I think It would also be OK with this version. Updated 👍